server/src/core/event/dto/create-event.input.ts

```diff
@@ -0,0 +1,10 @@
+import { ArgsType, Field } from '@nestjs/graphql';
+
+@ArgsType()
+export class CreateEventInput {
+  @Field({ description: 'Type of the event' })
```

Comment:
@FelixMalfait You can add input validation with `class-validator`, and example is available for the auth inputs